src/Application/Config.h

```diff
@@ -16,32 +16,31 @@
 //   limitations under the License.
 
 #include <string>
+#include "yaml-cpp/yaml.h"
```

Comment:
"" vs. <>, i.e. relative pathing vs. system here and L20

Reply:
Done!